lib/mastodon/media_cli.rb

```diff
@@ -113,5 +113,24 @@ def usage
       say("Imports:\t#{number_to_human_size(Import.sum(:data_file_size))}")
       say("Settings:\t#{number_to_human_size(SiteUpload.sum(:file_file_size))}")
     end
+
+    desc 'lookup', 'Lookup a toot by passing a media URL'
+    def lookup
+      prompt = TTY::Prompt.new
+
+      exit(1) unless (url = prompt.ask('Please enter a URL to the media to lookup:', required: true))
+
+      id = url
+           .split('/')[0..-2]
+           .select do |key|
+             key.scan(/\D/).empty?
```

Comment:
personally I've always found the \d / \D thing a little hard to keep track of, and besides that I think it unnecessarily inverts the logic here. what about just using a pattern like `\A\d+\Z`?

Reply:
Could you show me an example of how this might work in code? I'm not sure I follow how to do this with reject.